app/assets/stylesheets/header.scss

```diff
@@ -0,0 +1,191 @@
+$theme-colors: ( "primary": #fe8301);
+$primary-color:#fe8301;
+$body-bg: white;
+$secondary-color:#f2f2f2;
+$font: "Raleway", sans-serif;
+@media screen and (max-width: 1366px) {
+    .navbar-default{
+        background-color: $body-bg;
+    }
+}
+.container{
+  .navbar-brand{
+    img{
+      max-width: 40px;
+       margin-top: -7px;
+    }
+  }
+
+}
+.image-wrapper {
+    margin-top: 50px;
+    margin-bottom: 10px;
+}
+
+.image-hover:hover {
+    transform: scale(1.25);
+    z-index: 2;
+    .col-lg-3:not(:hover) {
+        opacity: 0.2;
+    }
+}
+
+.image-hover {
+    transform: scale(1);
+    transition: 0.3s ease;
+}
+
+nav.navbar {
+    padding-top: 0;
+    margin-bottom: 10px;
+    margin-top: 20px;
+    margin-right: 120px;
+    margin-left: 120px;
+}
+
+.navbar-custom {
+    .navbar-brand {
+        font-size: 12px;
+    }
+}
+
+h1,
+.navbar-brand,
+.navbar-nav a {
+    font-family: $font;
+    color: white;
+    font-size: 12px;
+    margin-right: 10px;
+    margin-left: 10px;
+    font-weight: 900;
+}
+
+#jumbotron-costum {
+    background-image: url("restaurante-cluj.jpg");
```

Comment:
In rails, use `image-url` instead of `url` in css. Read more [here](http://guides.rubyonrails.org/asset_pipeline.html#css-and-sass)

Also rename the images names in english